client/components/bulk-select/docs/example.jsx

```diff
@@ -57,7 +57,9 @@ module.exports = React.createClass( {
 					<a href="/devdocs/design/bulk-selects">BulkSelects</a>
 				</h2>
 				<Card>
-					<BulkSelect totalElements={ this.state.elements.length } selectedElements={ this.getSelectedElementsNumber() } onToggle={ this.handleToggleAll } />
+					<div>
```

Comment:
Since this is in the example code I am assuming that it is to just make it be enclosed in a block element. 
Which I think is perfectly fine. :+1: